pyqt5-gui/about.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+from PyQt5.QtWidgets import (
+    QVBoxLayout,
+    QHBoxLayout,
+    QPushButton,
+    QLabel,
+    QWidget,
+    QApplication
+)
+from PyQt5.QtGui import *
+from PyQt5.QtCore import *
+
+
+class About(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("About")
+        self.setGeometry(650, 300, 500, 200)
```

Comment:
Change the window size to 500/300pxs

Reply:
Resolved with [10c00e7](https://github.com/Stake-And-Rope/market-app/pull/33/commits/10c00e7b104d37b4baf95ac5e1c0f8f312cb755f)